safe-global/web-core
# fix: style tooltips via theme

## What it solves

Inconsistent `<Tooltip>` styles.

## How this PR fixes it

`<CustonTooltip>` has been removed in favour of overriding the global style of the MUI theme, matching [our design](https://www.figma.com/file/HKW5iarMST9rL7Kxkf59qY/MUI-for-Figma-Material-Pro-v5.9.0?node-id=6590%3A48770).

## How to test it

Hover over the "Batch Execute" button and delegate call warnings. Observe them being styled the same.

## Screenshots
![image](https://user-images.githubusercontent.com/20442784/186705887-0c853b91-5527-498a-8c40-cad79028eced.png)
![image](https://user-images.githubusercontent.com/20442784/186705896-eef4328d-61cb-4e21-a395-fb262ee4b37f.png)
